releases/release-5.0.6.md

```diff
@@ -0,0 +1,168 @@
+    (dup) - Fix the issue that CDC adds scan retries frequently due to the Congest error [#11082](https://github.com/tikv/tikv/issues/11082)
+    (dup) - Fix the issue that the Raft connection is broken when the channel is full [#11047](https://github.com/tikv/tikv/issues/11047)
+    - Fix panic caused by SST NotFound, which is ingested by  Lightning import. [#10438](https://github.com/tikv/tikv/issues/10438)
+    (dup) - Fix the issue that TiDB cannot correctly identify whether the `Int64` types in `Max`/`Min` functions are a signed integer or not, which causes the wrong calculation result of `Max`/`Min` [#10158](https://github.com/tikv/tikv/issues/10158)
+    - Fix follower meta corruption in rare cases with more than 4 replicas [#10225](https://github.com/tikv/tikv/issues/10225)
+    - Fix backup threads leak [#10287](https://github.com/tikv/tikv/issues/10287)
+    - copr cast invalid utf8 string to real bug fix [#23322](https://github.com/pingcap/tidb/issues/23322)
+
++ PD
+
+    (dup) - Fix a panic issue that occurs after the TiKV node is removed [#4344](https://github.com/tikv/pd/issues/4344)
+    - Fix the issue that operator can get blocked due to down store [#3353](https://github.com/tikv/pd/issues/3353)
+    (dup) - Fix slow leader election caused by stucked Region syncer [#3936](https://github.com/tikv/pd/issues/3936)
+    (dup) - Support that the evict leader scheduler can schedule Regions with unhealthy peers [#4093](https://github.com/tikv/pd/issues/4093)
+    - Fix the issue that there is the store limit of remove peer of the down store [#4090](https://github.com/tikv/pd/issues/4090)
+    - Fix the issue that the hot cache cannot be cleared when the heartbeat interval is less than 60 [#4390](https://github.com/tikv/pd/issues/4390)
+
++ TiFlash
+
+    - Fix potential data inconsistency after widening the integer primary key of a table
+    - Fix the issue that a TiFlash fails to start up on platforms without library "libnsl.so", like on ARM
+    - Fix the issue that "Store size" metric does not match the actual data size on disk
+    - Fix an issue that causes TiFlash crashing by "Cannot open file".
+    - Fix tiflash randomly crash when a mpp query is killed
+    - Fix the issue of unexpected error that "3rd arguments of function substringUTF8 must be constants"
+    - Increase the max supported depth of expression/plan tree in dag request from 100 to 200.
+    - Fix the bug that results of `where <string>` is wrong
+    - Fix the inconsistent behavior of `CastStringAsDecimal` between tiflash and tidb/tikv
+    - Fix the error "different types: expected Nullable(Int64), got Int64" which causes query failure
+    - Fix the error "Unexpected type of column: Nullable(Nothing)" which cases query failure
+    - Fix the issue that comparison between Decimal may cause overflow, which result in query failure
```

Comment:
```suggestion
    - Fix potential data inconsistency after widening the column type of an integer primary key
    - Fix the issue that TiFlash fails to start up on some platforms, such as ARM, due to the absence of library "libnsl.so"
    - Fix the issue that the "Store size" metric does not match the actual data size on a disk
    - Fix the issue that TiFlash crashea due to a "Cannot open file" error
    - Fix occasional crashes of TiFlash when an MPP query is killed
    - Fix the unexpected error "3rd arguments of function substringUTF8 must be constants"
    - Fix query failures caused by excessive `OR` conditions
    - Fix the bug that results of `where <string>` is wrong
    - Fix inconsistent behaviors of `CastStringAsDecimal` between TiFlash and TiDB/TiKV
    - Fix query failures caused by the error "different types: expected Nullable(Int64), got Int64"
    - Fix query failures caused by the error "Unexpected type of column: Nullable(Nothing)"
    - Fix query failures caused by overflow following Decimal comparison
```
"Increase the max supported depth of expression/plan tree in dag request from 100 to 200"----This bug fix looks more like an improvement. Have optimized the sentence. @flowbehappy. PTAL. Thanks.

Reply:
crashea -> crashed ?

Reply:
LGTM